events_lib_py/consumer/controller.py

```diff
@@ -0,0 +1,90 @@
+import logging
+from typing import Callable
+
+from confluent_kafka import Consumer, TopicPartition, OFFSET_BEGINNING
+
+from events_lib_py import config
+from events_lib_py.consumer import BaseKafkaConsumer
+from events_lib_py.dataclasses import EventHandlerResponse
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Controller(BaseKafkaConsumer):
+    entity = "controller"
+    gevent_pool_size = 100
+    attr_dataclass_map = {
+        "config": config.CONTROLLER_CONFIG,
+        "flag": config.CONTROLLER_FLAG,
+    }
+
+    def __init__(
+        self, config, attr_apply_handlers: dict[str, Callable[[int], None]] = {}
+    ):
+        super().__init__(config)
+        self.attr_apply_handlers = attr_apply_handlers
+        self.initial_run = True
+
+    def _subscribe_topic(self):
+        partitions = [
+            TopicPartition(
+                topic=self._config.controller_topic,
+                partition=0,
+                offset=OFFSET_BEGINNING,
+            )
+        ]
+        self._consumer.assign(partitions)
+        init_msg = self._consumer.poll(10)
```

Comment:
We could use defaults and exist with msg that dynamic config not defined and exit the thread